javamelody-core/src/main/java/net/bull/javamelody/internal/model/MBeansAccessor.java

```diff
@@ -168,4 +167,15 @@ private static Set<String> getAttributesNames(ObjectName name) {
 			return Collections.emptySet();
 		}
 	}
+    
+    private static boolean isMbeanAllocatedBytesEnabled() {
+        try {
+            Attribute supported = (Attribute) MBEAN_SERVER.getAttribute(THREADING, "ThreadAllocatedMemorySupported");
+            Attribute enabled = (Attribute) MBEAN_SERVER.getAttribute(THREADING, "ThreadAllocatedMemoryEnabled");
+            return ((boolean) supported.getValue()) && ((boolean) enabled.getValue());
+        } catch (Exception ex) {
+            // ignored
```

Comment:
ignored why ?

Reply:
No particular thing to do with this exception. But I accepted your suggestions anyway. Thanks a lot!